guides/common/modules/proc_configuring-the-project-mcp-server.adoc

```diff
@@ -0,0 +1,35 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-the-{project-context}-mcp-server"]
+= Configuring the {Project} MCP server
+
+Deploy the {Project} MCP server as a container and make it accessible to your MCP client.
+Perform these steps on the same system where your MCP client will run.
+
+.Procedure
+. Optional: Pull the latest version of the MCP container image from the registry:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ podman pull {mcp-server-image}
+----
+. Run the container, binding it to `localhost` and using the streamable HTTP transport mode:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ podman run --interactive --tty --publish localhost:8080:8080 \
+  {mcp-server-image} \
+  --foreman-url https://_{foreman-example-com}_ \
+  --host 0.0.0.0 \
+  --port 8080 \
+  --transport streamable-http
```

Comment:
```suggestion
  --foreman-url https://_{foreman-example-com}_
```

The last three options match the defaults https://github.com/theforeman/foreman-mcp-server/pull/13/files